server/app/views/questiontypes/StaticContentQuestionRenderer.java

```diff
@@ -51,7 +51,8 @@ public DivTag render(ApplicantQuestionRendererParams params) {
           TextFormatter.formatText(
               question.getQuestionText(),
               /* preserveEmptyLines= */ true,
-              /* addRequiredIndicator= */ false);
+              /* addRequiredIndicator= */ false,
+              /* ariaLabel= */ "opens in a new tab");
```

Comment:
This call is for admin, but it's the only situation where we need to preserve empty lines on the admin side so I opted to use the applicant method and pass in the aria label. If there were more situations on the admin side like this, it would warrant an admin specific method, but I don't think it's worth it for this one case.